### DISCUSSION TITLE
Different Sources with different data (trustworthiness)

### DISCUSSION THREAD

**Discussion by Parel803** [PygmalionOfCyprus/cmo-db-requests#92] [Q&A] [answered]
What if multiple sources give different data? Do I weight to source to the DB and give only one ref or can I give all data with corresponding sources and the DB manager decide the trustworthy of that source.

**Comment by PygmalionOfCyprus** [comment, marked as answer]
I'm just moving the answer from the OP into a comment so I can mark the thread as "answered" for future reference:

> I would make a call based on source trustworthiness, etc. and pick the number you feel is most reliable -- but also indicate that sources disagree and provide an explanation / links to contrasting sources in the comments/sources section. If nothing else, it demonstrates that you're aware of the other numbers and have made a conscious decision not to trust them (which will avoid cases of us thinking "Oh, this newer, etc. source disagrees, he probably just didn't see it...")
> 
> Worst case scenario we override your decision, but you'll at least have a chance to make your case.
